be/src/runtime/exec_env.h

```diff
@@ -239,6 +241,31 @@ class GlobalEnv {
     std::map<MemTrackerType, std::shared_ptr<MemTracker>> _mem_tracker_map;
 };
 
+class CacheEnv {
+public:
+    static CacheEnv* GetInstance();
+
+    Status init(const std::vector<StorePath>& store_paths);
+    void destroy();
+
+    void try_release_resource_before_core_dump();
+
+    BlockCache* block_cache() const { return _block_cache.get(); }
+    StoragePageCache* page_cache() const { return _page_cache.get(); }
+
+private:
+    Status _init_datacache();
+    Status _init_lru_base_object_cache();
+    Status _init_page_cache();
+
+    GlobalEnv* _global_env;
+    std::vector<StorePath> _store_paths;
+
+    std::shared_ptr<BlockCache> _block_cache;
+    std::shared_ptr<ObjectCache> _lru_based_object_cache;
+    std::shared_ptr<StoragePageCache> _page_cache;
```

Comment:
We will also add a `_starcache_based_object_cache` in CacheEnv in the future?

Reply:
Yes，will add next pr.